### ISSUE TITLE
context update should not require name

### ISSUE THREAD

**Issue by sirpy** [OrbisWeb3/orbis-sdk#9]
When trying to update a context access rules if you do not supply the `name` field the update request will fail.

**Comment by donatandelic** [comment]
This is expected/correct behavior, since Context's schema defines name as a non-nullable field.

You can check the latest Context schema commit on [Cerscan](https://cerscan.com/mainnet/stream/kjzl6cwe1jw147dp34t1t88xu2rfltlats6grzav216ko7ocwqz2fgq8myjw8gw).

**Comment by sirpy** [comment]
@donatandelic so when i do context update i need to pass all the previous fields values other wise they are going to be nullified?

**Comment by donatandelic** [comment]
@sirpy That's correct. You're updating the Ceramic stream directly, so whatever content you post as a part of the update will be treated as a new (fresh) commit. The chain will not try to amend or assume changes.
